src/main/java/org/spongepowered/mod/mixin/core/event/player/MixinEventPlayerInteractBlock.java

```diff
@@ -92,4 +96,13 @@ public Direction getSide() {
         return Optional.fromNullable(new Cause(null, this.entityPlayer, null));
     }
 
+    @Override
+    public void setClickedPosition(Vector3d clickedPosition) {
+        this.clickedPosition = Optional.of(VecHelper.toVector(this.pos).toDouble().add(clickedPosition));
```

Comment:
Not sure if this should be relative to the block or to the world. According to [git blame](https://github.com/SpongePowered/SpongeAPI/commit/57d70e3ccbde63583c3ee58bbf5059a249ac8e84#diff-458e8e0f271d0d47c8a82eb451d9d538R57) gabizou created the method. Which should it be @gabizou?